src/components/big-interactive-pages/editor.tsx

```diff
@@ -520,7 +521,7 @@ export default function Editor({ persistenceState, cookies, roomState }: EditorP
 			}
 		});
 	}, [initialCode]);
-	if(isNewSaveStrat.value && persistenceState.value.kind === "COLLAB" && typeof persistenceState.value.game === 'string')
+	if(isNewSaveStrat.value && persistenceState.value.kind === PersistenceStateKind.COLLAB && typeof persistenceState.value.game === 'string')
```

Comment:
While very simple and on a single line, this is the type of thing I'd wrap in a function w/ a good name, something that self-documents the decision you're trying to make.  TBH, even referring to this as "**_new_** save strat" isn't really great - it might be more helpful to refer to it as the "backend saving strategy" vs "frontend saving strategy".

I don't expect a change here, just thoughts for the future.